README.md

```diff
@@ -13,7 +13,7 @@ content is being installed on-cluster and how to create a running instance of th
 
 ## Components 
 
-RukPak is composed of a few primary APIs: `Bundle`, `Instance`, and `ProvisionerClass`
+RukPak is composed of a few primary APIs: `Bundle`, `BundleInstance`, and `Provisioner`
```

Comment:
nit: there's no Provisioner API.

Reply:
another nit: trailing period.

Reply:
yes correct, its not technically an API, but a concept worth describing here. Will update.